temp/OnnxTransformTempSolution/Microsoft.ML.Transforms.Onnx/OnnxTransform.cs

```diff
@@ -0,0 +1,294 @@
+                _outputColName = _outputNodeInfo.Name;
+                _host.Assert(_outputNodeInfo.DataType == _outputColType.ItemType.RawType);
+            }
+
+            public static OnnxMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISchema schema)
+            {
+                Contracts.CheckValue(env, nameof(env));
+                env.CheckValue(ctx, nameof(ctx));
+                ctx.CheckAtModel(GetVersionInfo());
+
+                byte[] modelBytes = null;
+                if (!ctx.TryLoadBinaryStream("OnnxModel", r => modelBytes = r.ReadByteArray()))
+                    throw env.ExceptDecode();
+                var model = new OnnxModel(modelBytes);
+
+                var inputColumn = ctx.LoadNonEmptyString();
+                var outputColumn = ctx.LoadNonEmptyString();
+                var transformArg = new Arguments() { InputColumn = inputColumn, OutputColumn = outputColumn };
+
+                return new OnnxMapper(env, schema, transformArg, model);
+            }
+
+            public RowMapperColumnInfo[] GetOutputColumns()
+            {
+                var info = new RowMapperColumnInfo[1];
+                info[0] = new RowMapperColumnInfo(_outputColName, _outputColType, null);
+                return info;
+            }
+
+            public Func<int, bool> GetDependencies(Func<int, bool> activeOutput)
+            {
+                return col => activeOutput(0) && (_idvToTensorAdapter._idvColumnIndex == col);
+            }
+
+            public void Save(ModelSaveContext ctx)
+            {
+                _host.AssertValue(ctx);
+                ctx.CheckAtModel();
+                ctx.SetVersionInfo(GetVersionInfo());
+
+                ctx.SaveBinaryStream("OnnxModel", w => { w.WriteByteArray(_model.ToByteArray()); });
+                ctx.SaveNonEmptyString(_args.InputColumn);
+                ctx.SaveNonEmptyString(_args.OutputColumn);
+            }
+
+            public Delegate[] CreateGetters(IRow input, Func<int, bool> activeOutput, out Action disposer)
+            {
+                disposer = null;
+                var getters = new Delegate[1];
+                using (var ch = _host.Start("CreateGetters"))
+                {
+                    if (activeOutput(0))
+                        getters[0] = Utils.MarshalInvoke(MakeGetter<int>, _outputNodeInfo.DataType, input);
+
+                    ch.Done();
+                    return getters;
+                }
+            }
+
+            private Delegate MakeGetter<T>(IRow input)
+            {
+                _host.AssertValue(input);
+                _host.Assert(typeof(T) == _outputColType.ItemType.RawType);
+
+                _idvToTensorAdapter.InitializeValueGetters(input);
+
+                ValueGetter<VBuffer<T>> valuegetter = (ref VBuffer<T> dst) =>
+                {
+                    var outputTensors = _model.Run(new List<Tensor> { _idvToTensorAdapter.GetTensor() });
```

Comment:
>_model [](start = 40, length = 6)

Does Microsoft.ML.Scoring take care of disposing this object? Otherwise, we should add it to the disposer in CreateGetters.